vestido/vestido-storefront/modules/Cart/CartView.tsx

```diff
@@ -296,22 +304,60 @@ const CartView: React.FC = () => {
             </div>
           </div>
 
-          <div className="md:basis-1/3 flex flex-col">
-            <div className="bg-gray-100 p-10 min-h-[275px] relative flex flex-col justify-between">
+          <div className="md:basis-1/3 overflow-auto  px-3 md:pl-5 md:sticky top-0 w-full text-white">
+            <div className="flex flex-col">
               <div>
-                <div className="flex items-center text-neutral-800 justify-center pb-3 font-semibold text-xl justify-between ">
-                  Items Total: <div>₹&nbsp;{totalPrice.toFixed(2)}</div>
-                </div>
+                <Table>
+                  <TableHeader>
+                    <TableRow className="border-none hover:bg-transparent">
+                      <TableHead>Item</TableHead>
+                      <TableHead>Qty</TableHead>
+                      <TableHead className="text-right">Amount</TableHead>
+                    </TableRow>
+                  </TableHeader>
 
-                <div className="font-medium text-[8px] md:text-sm">
-                  Shipping charges calculated at checkout (Free Shipping all
-                  over Kerala)
-                </div>
+                  <TableBody>
+                    {cartItems?.map((cartItem, index) => {
+                      const itemAmount = (
+                        cartItem.item.price * cartItem.qty
+                      ).toFixed(2);
```

Comment:
Please don't do any kind of calculations from the client side. Make use of the data coming from `useCalculateTotals`